packages/forms/src/UIForm/fields/NestedListView/NestedListView.test.js

```diff
@@ -68,6 +68,36 @@ describe('NestedListView component', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	describe('componentDidUpdate', () => {
+		it('should update items on props.value change', () => {
+			// given
+			const node = document.createElement('div');
+			// eslint-disable-next-line react/no-render-return-value
+			const instance = ReactDOM.render(<NestedListViewWidget {...props} value={{}} />, node);
```

Comment:
Pretty interesting to see this eslint rule disabled: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-render-return-value.md
It would make the test less readble to fix the lint error

Reply:
I find it more readable too  
Do we keep it like this ?